src/app/app-routing.module.ts

```diff
@@ -1,7 +1,16 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { AboutMePageComponent } from './components/about-me-page/about-me-page.component';
+import { CareerPageComponent } from './components/career-page/career-page.component';
+import { LandingPageComponent } from './components/landing-page/landing-page.component';
+import { ProjectsPageComponent } from './components/projects-page/projects-page.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', component: LandingPageComponent},
+  { path: 'About Me', component: AboutMePageComponent},
```

Comment:
I would remove the space in the path name to avoid any funkiness

Reply:
Done